src/integration/java/io/pinecone/integration/controlPlane/pod/ConfigureIndexTest.java

```diff
@@ -138,4 +144,21 @@ public void sizeIncrease() throws InterruptedException {
             assertEquals(podSpec.getPodType(), "p1.x2");
         });
     }
+
+    private static void waitUntilIndexStateIsReady(String indexName) throws InterruptedException {
+        int timeToWaitMs = 30000;
+        int timeWaited = 0;
+        IndexModel index = controlPlaneClient.describeIndex(indexName);
+
+        while (index.getStatus().getState() != IndexModelStatus.StateEnum.READY && timeWaited <= timeToWaitMs) {
+            Thread.sleep(2000);
+            timeWaited += 2000;
+            System.out.println("waited 2000ms for index to upgrade, time left: " + timeToWaitMs);
```

Comment:
My only addition to this PR would be to turn the print stmts into logging (info) statements so that this dat is useful to those in production who likely aren't looking at the console

Reply:
Right, agreed thank you! Updated.